CleanProFinder/CleanProFinder.Server/Controllers/AccountController.cs

```diff
@@ -39,5 +39,14 @@ public AccountController(IMediator mediator, IMapper mapper)
             var result = await _mediator.Send(request, cancellationToken);
             return ConvertFromServiceResponse(result);
         }
+
+        [HttpPost("service-provider/account/create")]
```

Comment:
please change endpoint to new format account/service-provider/create (initial created by me was wrong) and add docs with example of user signup

Reply:
fixed